lib/ruby_lsp/store.rb

```diff
@@ -14,20 +14,19 @@ def initialize
       @state = T.let({}, T::Hash[String, Document])
     end
 
-    sig { params(uri: String).returns(Document) }
+    sig { params(uri: String).returns(T.nilable(Document)) }
```

Comment:
Why is this `nilable` now? It looks like `set` will still always store a document?